docs/release-notes/RELEASE-NOTES-3.0.0-rc.1.md

```diff
@@ -3,13 +3,62 @@
 [This][this-release] is the first Fortuna release candidate.
 It marks the freeze of the Fortuna consensus protocol and of the user API.
 
-It:
-* Does all the things mentioned temporarily in files [/docs/release-notes/next/PT-*.md](/docs/release-notes/next/).
-* Does all the things mentioned temporarily in files [/docs/release-notes/next-fortuna/PT-*.md](/docs/release-notes/next-fortuna/).
-
-TODO: When preparing the release, concatenate all `/docs/release-notes/next/*` Markdown files and place them in this file. (Hint: you can use auxiliary script `scripts/cat-files-in-directory-sorted-by-committer-date` and command `git log -p -w --color-moved`.)
-
-TODO: When preparing the release, concatenate all `/docs/release-notes/next-fortuna/*` Markdown files and place them in this file. (Hint: you can use auxiliary script `scripts/cat-files-in-directory-sorted-by-committer-date` and command `git log -p -w --color-moved`.)
+Please refer to the notes below for details and backward compatibility.
+
+Regarding the Fortuna consensus protocol upgrade in testnet, this release:
+* Sets the Fortuna testnet hard fork height to block 82900 (approximately Mon 20th May 2019 at 3:30am CEST).
+
+Regarding the Block Reward Initiative, this release:
+* Introduces a mechanism to split mining reward and send to predefined address. This is consensus breaking.
+
+Regarding introduction of generalized accounts from the Fortuna consensus protocol, this release:
+* Adds generalized accounts. See
+  [here](https://github.com/aeternity/protocol/blob/aeternity-node-v3.0.0/generalized_accounts/generalized_accounts.md)
+  for a description of the new feature.
+* Changes the HTTP API for /transactions/info
+  This HTTP endpoint used to just return a contract call object, but now returns <<"call_info">> => ContractCallObject instead, i.e. an extra tag is added
```

Comment:
```suggestion
  This HTTP endpoint used to just return a contract call object, but now returns <<"call_info">> => ContractCallObject instead, i.e. an extra tag is added.
```

Reply:
Markdown discards the newline in the bullet list so a dot there improves readability a bit.